bokeh/util/tests/test_forcing.py

```diff
@@ -0,0 +1,61 @@
+from numpy.testing import assert_allclose
+
+import bokeh.util.forcing as forcing
+
+def _collector(results):
+    def foo(val):
+        results.append(val)
+    return foo
+
+w = 0.3
+A = 3
+phi = 0.1
+offset = 2
+
+def test_sine():
```

Comment:
so my convention is to have `test_<modulename>` and so adding `forcing` to `test_sine` inside `test_forcing` seems redundant to me. It's testing `forcing.sine` by virtue of the fact that it is in `test_forcing.py`

For the other, I'd prefer `_sequence_values`, which is what I will add.